assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -65,8 +68,16 @@ function DashboardSummaryWidget() {
 			metrics,
 		};
 
+		// Get the first day of the month an ISO 8601 date string without the time.
+		const startOfMonth = new Date(
+			new Date( referenceDate ).getFullYear(),
+			new Date( referenceDate ).getMonth(),
+			1
+		).toISOString().substr( 0, 10 );
```

Comment:
We already have a utility for formatting a `Date` into our `YYYY-MM-DD` format, which we should use here instead of the ISO+substring approach. See:
https://github.com/google/site-kit-wp/blob/37c5b0456cd9aa8545e139a0adbc8b6aa7102039/assets/js/googlesitekit/datastore/user/utils/get-date-string.js#L38

This is the same utility used by our datastore date range selectors, so as long as we use this utility, we know that it will be formatted correctly.

While this is currently a utility of the `core/user` datastore, it should really be a generic date utility, so I'll open a new issue for moving these.

Reply:
Aforementioned new issue: https://github.com/google/site-kit-wp/issues/2641

Reply:
Thanks @aaemnnosttv, this makes sense to me. I've added the blocking association with this ticket.

Reply:
I resolved this using the existing location of the helper, it can be updated later in #2641